p2p/peer.go

```diff
@@ -37,6 +41,7 @@ type peerConn struct {
 	persistent bool
 	config     *PeerConfig
 	conn       net.Conn // source connection
+	ips        []net.IP
```

Comment:
are you thinking we'll use this for something ? looks like we only ever use the [0]